models/intel/image-retrieval-0001/model.yml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  `image-retrieval-0001` is an image retrieval model based on Mobilenet v2 as a backbone. The model produces l2-normalized embeddings with dimension equals to 256.
+task_type: "object_attributes"
```

Comment:
Please remove the quotation marks (from here and other entries), for consistency with other models.

Reply:
done